quick_tour/the_view.rst

```diff
@@ -160,7 +160,7 @@ Don't forget to check out the official `Twig documentation`_ to learn everything
 about filters, functions and tags.
 
 Including other Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------
```

Comment:
I propose this change because I don't see why the content under title "Including other Templates" (and "Embedding other Controllers") would be subtitle of the title "Using Tags, Filters and Functions" based on the content. If this decision is based on the fact that ``include()``, ``render()`` or ``controller()`` are Twig's functions then also other titles like "Creating Links between Pages" and so one should be subtitles of the title "Using Tags, Filters and Functions".